nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -119,6 +119,12 @@ CACHEOPS_DEFAULTS = {"timeout": int(os.getenv("NAUTOBOT_CACHEOPS_TIMEOUT", 900))
 # Set to False to disable caching with cacheops. (Default: True)
 CACHEOPS_ENABLED = is_truthy(os.getenv("NAUTOBOT_CACHEOPS_ENABLED", True))
 
+# Celery broker URL used to tell workers where queues are located
+CELERY_BROKER_URL = os.getenv("NAUTOBOT_CELERY_BROKER_URL", CACHES["default"]["LOCATION"])
+
+# Celery results backend URL to tell workers where to publish task results
+CELERY_RESULT_BACKEND = os.getenv("NAUTOBOT_CELERY_RESULT_BACKEND", CACHES["default"]["LOCATION"])
+
 # Global Celery task timeout (in seconds)
 CELERY_TASK_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", 30 * 60))
```

Comment:
This should probably be removed from the template as unnecessary/redundant with settings.py.

Reply:
Yeah, that's in line w/ that. We should really defer to having only the required settings in the template config, and having everything else fall back to environment variables in the core settings, as @nniehoff had recently reminded us, too.